supplier_management_site/tests/test.py

```diff
@@ -1,13 +1,23 @@
-from django.test import SimpleTestCase, RequestFactory
-from django.utils.translation import activate, ugettext_lazy as _
 from parameterized import parameterized
-from .views import home
 
+from django.core import mail
+from django.test import TestCase, RequestFactory
+from django.utils.translation import activate, ugettext_lazy as _
+
+from supplier_management_site.tests.views import (
+    home,
+)
 
-class HomePageTests(SimpleTestCase):
+from utils.send_email import send_email_notification
+
+
+class TestTranslationConfiguration(TestCase):
     def setUp(self):
         self.factory = RequestFactory()
 
+    def tearDown(self):
+        activate('en')
+
```

Comment:
Why did you add this tearDown?

Reply:
The test is activating portuguese as las language and is breaking other tests so i tear down reestablishing english as default lang.